### PR TITLE
zoom: fixes xcb-utils-* dependencies

### LINKED ISSUE #26844: zoom has missing dependencies
Zoom requires the following:

- `xcb-util-image`
- `xcb-util-keysyms`

Solution taken from: https://www.reddit.com/r/voidlinux/comments/hkqks4/building_zoom_on_void/

### System

* xuname:  
  *Void 5.8.18_1 x86_64 GenuineIntel uptodate rFFF*
* package:  
  *affected package(s) including the version*: ``zoom-5.4.54779.1115_1``

### Expected behavior

No need to install dependencies.

### Actual behavior

I need to install these in order to make the application run.

### Steps to reproduce the behavior
Download void-packages, enable restricted, build `zoom`, install with `xu zoom` from directory `void-packages/hostdir/binpkgs/nonfree/`

### PR DESCRIPTION
**Pull request by waynevanson**
closes #26844

### CODE REVIEW

**Comment by mobinmob** [comment]
Please bump revision ;)

File: srcpkgs/zoom/template [hunk 1]
```diff
@@ -5,7 +5,7 @@ revision=1
 archs="x86_64"
 wrksrc=zoom
 create_wrksrc=yes
-depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative')"
+depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative') xcb-util-image xcb-util-keysyms"
```
Comment (fosslinux) [inline srcpkgs/zoom/template:8]: split into a new line, it's ok to have it over 80 chars for the vopt_if bit i guess but please split the new dependencies onto a new line.

**Review by fosslinux — CHANGES_REQUESTED** [review]

**Comment by waynevanson** [comment]
@mobinmob did I do the bump for the revision correctly?

File: srcpkgs/zoom/template [hunk 2]
```diff
@@ -1,11 +1,14 @@
 # Template file for 'zoom'
 pkgname=zoom
-version=5.4.54779.1115
+version=5.4.54779.1115_1
 revision=1
```
Comment (ericonr) [inline srcpkgs/zoom/template]:
```suggestion
version=5.4.54779.1115
revision=2
```

**Comment by waynevanson** [comment]
Apologies for multiple force pushes, forgot to `git add .` before `git commit --amend`.

File: srcpkgs/zoom/template [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
-revision=1
+revision=2
 archs="x86_64"
 wrksrc=zoom
 create_wrksrc=yes
-depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative')"
+depends="
+	$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative')
+	xcb-util-image xcb-util-keysyms
+"
```
Comment (ericonr) [inline srcpkgs/zoom/template:11]:
```suggestion
depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative')
 xcb-util-image xcb-util-keysyms"
```

We indent multi-line variables with spaces

**Pull request merged**